ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -1,75 +1,51 @@
 Library             ../../../../libs/DataSciencePipelinesAPI.py
 Suite Setup         Data Science Pipelines Suite Setup
 Suite Teardown      RHOSi Teardown
 
 
 *** Variables ***
-${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
-${DATA_SCIENCE_PIPELINES_APPLICATION_YAML}=    ods_ci/tests/Resources/Files/data-science-pipelines-sample.yaml
 ${URL_TEST_PIPELINE_RUN_YAML}=                 https://raw.githubusercontent.com/opendatahub-io/data-science-pipelines-operator/main/tests/resources/dsp-operator/test-pipeline-run.yaml
-&{OPENSHIFT_PIPELINES_VERSION}=                4.10=pipelines-1.8   4.11=pipelines-1.9  4.12=pipelines-1.10   4.13=pipelines-1.10  #robocop:disable
+
 
 *** Test Cases ***
-Verify ODS users can create and run a data science pipeline using the API
+Verify Ods Users Can Create And Run a Data Science Pipeline Using The Api
     [Documentation]    Creates, runs pipelines with admin and regular user. Double check the pipeline result and clean
     ...    the pipeline resources.
     [Tags]      Sanity
     ...         Tier1
     ...         ODS-2083
-    End To End Pipeline Workflow Via Api    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    myproject1
-    End To End Pipeline Workflow Via Api    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    myproject2
+    End To End Pipeline Workflow Via Api    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    pipelinesapi1
+    End To End Pipeline Workflow Via Api    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    pipelinesapi2
 
 
 *** Keywords ***
 End To End Pipeline Workflow Via Api
-    [Documentation]    Create, run and double check the pipeline result. In the end, clean the pipeline resources.
+    [Documentation]    Create, run and double check the pipeline result using API. In the end, clean the pipeline resources.    # robocop: disable:line-too-long
     [Arguments]     ${username}    ${password}    ${project}
     Remove Pipeline Project    ${project}
-    Create Pipeline Project    ${project}
+    New Project    ${project}
     Install DataSciencePipelinesApplication CR    ${project}
     ${status}    Login Using User And Password    ${username}    ${password}    ${project}
     Should Be True    ${status} == 200    DSP routing is working
     ${pipeline_id}    Create Pipeline    ${URL_TEST_PIPELINE_RUN_YAML}
     ${run_id}    Create Run    ${pipeline_id}
     ${run_status}    Check Run Status    ${run_id}
     Should Be True    '${run_status}' == 'Completed'    Run ends
-    [Teardown]    Clear Data Science Pipelines Resources    ${run_id}    ${pipeline_id}    ${project}
-
-Clear Data Science Pipelines Resources
-    [Documentation]    Cleans the pipeline resources.
-    [Arguments]     ${run_id}    ${pipeline_id}    ${project}
     Delete Runs    ${run_id}
     Delete Pipeline    ${pipeline_id}
-    Remove Pipeline Project    ${project}
+    [Teardown]    Remove Pipeline Project    ${project}
```

Comment:
Teardown must be in the Test Case, not in the keyword

Reply:
@diegolovison if you added `Remove Pipeline Project` in the keyword teardown to ensure that the project was deleted even if there was a problem deleting the runs o pipeline, that shouldn't be needed because all the keywords of Clear Data Science Pipelines Resources will be run even if one of them fails. The reason is because continue on failure is automatically enabled for Teardown keywords. More info at https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#continuing-on-failure